candig/server/cli/repomanager.py

```diff
@@ -2589,7 +2589,7 @@ def repo_main(args=None):
         # Uncaught exception: this is a bug
         message = """
 An internal error has occurred.  Please file a bug report at
-https://github.com/ga4gh/ga4gh-server/issues
+https://github.com/can dig/candig-server/issues
```

Comment:
It is not proper url, is not it? Or it is just my screen rendering...

Reply:
Ah yes, that's not a proper url..